src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,22 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int ELIGIBLE_AGE = 35;
+    private static final int INDEX_OF_AGE_FROM = 0;
+    private static final int INDEX_OF_AGE_TO = 1;
+    private static final int ELIGIBLE_PERIOD = 10;
+    private static final String ELIGIBLE_NATIONALITY = "Ukrainian";
+
+    @Override
+    public boolean test(Candidate candidate) {
+        String[] temp = candidate.getPeriodsInUkr().split("-");
```

Comment:
I am not sure , but maybe you should add some constant for this like DATA_DELIMITER or something else
```suggestion
        String[] temp = candidate.getPeriodsInUkr().split(SOME_CONSTANT);
```